StickyTabBarViewController/Classes/ExpandableViewController.swift

```diff
@@ -88,10 +88,7 @@ class ExpandableViewController: UIViewController {
     private func configureChildVC() {
         addChild(childVC)
         view.addSubview(childVC.view)
-        childVC.view.topAnchor.constraint(equalTo: view.topAnchor, constant: 0.0).isActive = true
-        childVC.view.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0.0).isActive = true
-        childVC.view.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0.0).isActive = true
-        childVC.view.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 0.0).isActive = true
+        childVC.view.frame = view.bounds
```

Comment:
Cool! 
So I guess the conflicting constraints where coming from the ones already set by the system & yours? 

Meaning the `translatesAutoresizingMaskIntoConstraints` was adding those conflicting ones?

Reply:
Yes I believe so, now it should be fine regardless how child vc's constraints are set @Thurman1776-

Reply:
> > I increased version to 0.0.5 but not sure if I should make a release and update readme. It is kind of a critical fix but, pod is not mature enough to be used yet anyways, what do you think?
> 
> Didn't we agree on publishing this pod as a stable version once this issue was fixed? 🤔
> Can't really recall as it is been a while 😬
> @emrepun

I think we did yes 😅So then I should increase version to 1.0.5 and go through README to see if all is fine and nothing is missing before publishing it? @Thurman1776-